src/cgutils.cpp

```diff
@@ -4485,6 +4485,12 @@ static int compare_cgparams(const jl_cgparams_t *a, const jl_cgparams_t *b)
 }
 #endif
 
+static jl_cgval_t emit_memorynew(jl_codectx_t &ctx, jl_datatype_t *mty_dt, jl_cgval_t len)
+{
+    // Jameson HALP
```

Comment:
@vtjnash can I get some help on writing this part?

Reply:
Is this a special case where mty_dt is const?

Reply:
no (although it might be worth special casing that)